test/manual/definitions/apm/.net/linux/ubuntu20-aspnetcore.json

```diff
@@ -0,0 +1,50 @@
+{
+    "global_tags": {
+        "owning_team": "OpenSource",
+        "Environment": "development",
+        "Department": "Product",
+        "Product": "Virtuoso"
+    },
+
+    "resources": [{
+        "id": "ubuntunetcore1",
+        "display_name": "NetCoreUbuntu20Host",
+        "provider": "aws",
+        "type": "ec2",
+        "size": "t3.nano",
+        "ami_name": "ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-????????.1",
+        "user_name": "ubuntu"
+    }],
+
+    "services": [{
+        "id": "aspnetcore",
+        "destinations": ["ubuntunetcore1"],
+        "source_repository": "https://github.com/newrelic/open-install-library.git",
+        "deploy_script_path": "test/deploy/linux/.net/install/ubuntu20/roles",
+        "port": 9998
```

Comment:
9999 (which is not what is used here, but close) is a special value for the deployer, could we update the port value to be similar to the other 2 services in the 8xxx range

Reply:
The `aspnetcore` "service" just installs the .NET runtime, so the port number is arbitrary.

Reply:
Ok, so if we don't need that specific port to be opened on the host, I would suggest using the reserved value of 9999. This tells the deployer not to open that port to the public when creating the ec2.

Reply:
Done!